test/topology/test_start_bootstrapped_with_invalid_seed.py

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+import asyncio
+import pytest
+import time
+import logging
+
+from test.pylib.internal_types import IPAddress
+from test.pylib.manager_client import ManagerClient
+from test.pylib.util import wait_for_cql_and_get_hosts, wait_for_cql
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.asyncio
+async def test_raft_snapshot_truncation(manager: ManagerClient):
+    """
+    Check that when bootstrapped, having a non-resolving DNS name in
+    the config, does prevent node restart.
+    Issue https://github.com/scylladb/scylladb/issues/14945.
+
+     1. Start 3 nodes.
+     2. Wait till all start and join the cluster.
+     3. Stop one node & and start it with a non-existing name in the seeds list config.
+     4. Make sure the node started successfully since it's already a cluster member.
+
+    """
+    servers = await manager.servers_add(3)
+    cql = manager.get_cql()
+
+    s1 = servers[0]
+    await wait_for_cql_and_get_hosts(cql, [s1], time.time() + 60)
```

Comment:
The test description says "Wait till all start", however I think that this line only waits for the `s1` server to start? Could as well pass all servers here to wait for all of them to start ...

Reply:
`servers_add` waits for servers to start; this line only checks that the driver session under `cql` has connectivity to `s1`.

Which by the way is not necessary in this test since we're not performing any operations with `cql`.

Reply:
As far as I understand `await manager.servers_add(3)` should start the servers and wait for them to start: 
`await self.driver_connect(**driver_connect_opts)` at manager_client.py:386